pages/api/getLandOffers.js

```diff
@@ -0,0 +1,23 @@
+/**
+ *
+ * @param {*} req
+ * @param {*} res
+ */
+export default async function handler(req, res) {
+  let metaverse = req.body.metaverse;
+  let tokenID = req.body.tokenId;
+  const url = `https://api.opensea.io/api/v1/asset/${metaverse}/${tokenID}/offers`;
+  try {
+    const response = await fetch(url, {
+      method: "GET",
+      headers: {
+        "Content-Type": "application/json",
+        "X-API-KEY": process.env.NEXT_PUBLIC_OPENSEA,
+      },
+    });
+    const data = await response.json();
+    res.json(data);
+  } catch (err) {
+    res.status(400).json(err);
+  }
```

Comment:
All good! But try to make this as a typescript file as well. you can check the file `[tokenId].ts` for how to do it. Also on how to turn this so that you use the `req.query.metaverse` instead of `req.body.metaverse`

Reply:
And also for `Axie` like mentioned before. You will have to retrieve that data from the axie marketplace. I think I sent you a link to their API on slack some days ago.